types/address_test.go

```diff
@@ -567,3 +567,13 @@ func (s *addressTestSuite) TestGetFromBech32() {
 	s.Require().Error(err)
 	s.Require().Equal("invalid Bech32 prefix; expected x, got cosmos", err.Error())
 }
+
+func (s *addressTestSuite) TestMustAccAddressFromBech32() {
```

Comment:
can you fix this test?

Reply:
The prefix dynamically, so the address cannot be fixed. This unit test has been fixed